wiki/FitNesseRoot/HsacAcceptanceTests/SlimTests/BrowserTest/DoubleClickByLabelOrButton/content.txt

```diff
@@ -0,0 +1,32 @@
+This test ensures that we can double click on buttons, based on their displayed text, and on labels, even if the associated input is invisible.
```

Comment:
I believe the test now (correctly) only checks we can double click. Let's have it state that